app/src/androidTest/java/com/okta/idx/android/cucumber/definitions/DashboardDefinitions.kt

```diff
@@ -117,6 +119,27 @@ class DashboardDefinitions {
         onView(withId(R.id.sign_out_button)).perform(scrollTo()).perform(click())
     }
 
+    @And("^she sees a table with her profile info$")
+    fun she_sees_a_table_with_her_profile_info() {
+        onView(withId(R.id.claims_linear_layout))
+            .check(matches(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
+    }
+
+    @And("^the cell for the value of \"email\" is shown and contains her email$")
+    fun the_cell_for_the_value_of_email_is_shown_and_contains_her_email() {
+        Thread.sleep(1000)
```

Comment:
Haven`t you previously added some method to wait for view?

Reply:
Yeah, this is definitely a hack. I can follow up with a better approach.